pages/crafts/crafts.html

```diff
@@ -365,7 +372,7 @@ <h6 class="mb-3">
           <div class="modal fade" id="intro-modal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
             <div class="modal-dialog modal-xl  modal-dialog-centered" role="document">
               <div class="modal-content" style="background: none;border: none;">
-                <div class="modal-body" style="border-radius: 5px;margin-top: 50px;padding: 0;">
+                <div class="modal-body" style="border-radius: 5px;margin-top: 5vh;padding: 0;" id="yt-player">
 
                   <div class="embed-responsive embed-responsive-16by9">
                     <iframe class="embed-responsive-item" style="height: 70vh;width: 100%;" src="https://www.youtube.com/embed/zBf7DOtWRP8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Bro, the video's is not loading (on local page):

![image](https://user-images.githubusercontent.com/38415384/123818317-bc98bb80-d8ad-11eb-878e-2ccc9fd24086.png)

Try with this link https://www.youtube.com/watch?v=Jwgf3wmiA04

Reply:
yeah same its the problem on the youtube side

Reply:
they have some policy preventing us from running in local so.

Reply:
oh.. okay.. so shall I merge this.. everything else look fine for now..

Reply:
yes